Doc/source/designspaceLib/readme.rst

```diff
@@ -1044,6 +1044,19 @@ Recommendation for editors
 -  Use ``documentObject.updateFilenameFromPath()`` to explicitly set the
    **filename** attributes for all instance and source descriptors.
 
+Common Lib Key Registry
```

Comment:
Formatting niggles, I think this should bump up to the level of a heading, be given the number 7, and have the `7 This document` change to `8 This document`